easybuild/easyconfigs/g/Geant4/Geant4-10.6-intel-2019b.eb

```diff
@@ -0,0 +1,41 @@
+name = 'Geant4'
+version = '10.6'
+
+homepage = 'http://geant4.cern.ch/'
+
+description = """
+ Geant4 is a toolkit for the simulation of the passage of particles through
+ matter.  Its areas of application include high energy, nuclear and accelerator
+ physics, as well as studies in medical and space science.
+"""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+
+source_urls = ['http://geant4.cern.ch/support/source']
+sources = ['%(namelower)s.%(version_major)s.0%(version_minor)s.tar.gz']
+checksums = ['1424c5a0e37adf577f265984956a77b19701643324e87568c5cb69adc59e3199']
+
+builddependencies = [
+    ('CMake', '3.15.3'),
+]
+
+dependencies = [
+    # recommended CLHEP version, see http://geant4-data.web.cern.ch/geant4-data/ReleaseNotes/ReleaseNotes4.10.6.html#2.
+    ('CLHEP', '2.4.1.3'),
+    ('expat', '2.2.7'),
+    ('libGLU', '9.0.1'),
+    ('Mesa', '19.1.7'),
+    ('Qt5', '5.13.1'),
+    ('X11', '20190717'),
+    ('Xerces-C++', '3.2.2'),  # for GDML
+]
+
+configopts = ''
+configopts += ' -DGEANT4_BUILD_MULTITHREADED=ON'
+configopts += ' -DGEANT4_INSTALL_DATA=ON -DGEANT4_INSTALL_DATADIR=%(installdir)s/../data'
```

Comment:
@JackPerdue Current easyconfigs use `-DGEANT4_INSTALL_DATA=OFF`, why change this?

Reply:
Try 'source'ing $EBROOTGEANT4/share/Geant4-10.6.0/geant4make/geant4make.sh if you use DATA=OFF.... one of our users did and wasn't happy.  SLAC suggested %(installdir)s/../data  so it could be shared among different versions.  Works much better now for those wanting to use geant4make.sh (or at least doesn't throw  a lot of errors).

Reply:
(kinda like FOAM_BASH for OpenFOAM (which still has a broken WM_CC last time I checked.... years after pointing out the issue).... nobody cares until they need to build something with it)

Reply:
I understand your motivation for this, but I'm still not sure this is a good idea...

The `data` directory is a whopping 3.7GB (~40k files), which typically isn't a very good fit for a filesystem hosting software installations.

Moreover, parking the `data` in the parent directory to make it sharable between different versions doesn't sound like a very good idea either to me... Will this data never change? What if another version is installed, will it blindly overwrite the data, or update it? Who is SLAC that made this recommendation?

Can we somehow signal that the data must be downloaded first to a well suited location, before this `Geant4` installation is used?

Reply:
Possibly related: https://github.com/easybuilders/easybuild-easyblocks/issues/1352